packages/insomnia/src/ui/routes/design.tsx

```diff
@@ -197,7 +197,7 @@ const Design: FC = () => {
   const updateApiSpecFetcher = useFetcher();
   const generateRequestCollectionFetcher = useFetcher();
   const [isLintPaneOpen, setIsLintPaneOpen] = useState(false);
-  const [isSpecPaneOpen, setIsSpecPaneOpen] = useState(true);
+  const [isSpecPaneOpen, setIsSpecPaneOpen] = useState(false);
```

Comment:
Based on the details of the ticket I think we should check if the spec has contents and decide on what the default state is.
```suggestion
  const [isSpecPaneOpen, setIsSpecPaneOpen] = useState(Boolean(parsedSpec));
```